docs/sphinx/manual/hpx_runtime_and_resources.rst

```diff
@@ -184,13 +185,33 @@ thread pool.
 Using the resource partitioner
 ------------------------------
 
+Before |hpx| 1.5.0
+~~~~~~~~~~~~~~~~~~
+
+This version is outdated from 1.5.0 onwards, the resource partitioner is now
+created during |hpx| runtime initialization. Please refer to :ref:`resource
+partitioner after 1.5.0 onwards <_using_rp_from_1_5_0>`. We keep the
+documentation to show the difference between the two versions.
+
 In order to create custom thread pools the resource partitioner needs to be set
 up before |hpx| is initialized by creating an instance of
 :cpp:class:`hpx::resource::partitioner`:
 
-.. literalinclude:: ../../libs/resource_partitioner/examples/simplest_resource_partitioner_1.cpp
-   :start-after: //[body
-   :end-before: //body]
+.. code-block:: c++
```

Comment:
How come this is inlined? Does the `literalinclude` not work?

Reply:
We changed the examples to use the new way, so to put the old way, I just put raw code as we don't have examples for that, the corresponding `literalinclude` can be found in the next section which explains the new way of doing things

Reply:
Those old examples don't exist anymore if I'm not mistaken, but I'll rework this whole section..

Reply:
Yep, my bad. Sorry for the noise.